Sources/ToastView.swift

```diff
@@ -71,6 +71,10 @@ open class ToastView: UIView {
     @unknown default: return 20
     }
   }()
+  
+  /// If this value is `true` and SafeArea is available,
+  /// `safeAreaInsets.bottom` will be added to the `bottomOffsetPortrait` and `bottomOffsetLandscape`.
+  @objc open dynamic var useSafeAreaForBottomOffset: Bool = true
```

Comment:
Should be false by default:
- avoiding changing the existing behavior,
- the home bar is often hidden anyway.